modules/javafx.controls/src/main/java/javafx/scene/chart/BarChart.java

```diff
@@ -61,6 +61,8 @@
  * A chart that plots bars indicating data values for a category. The bars can be vertical or horizontal depending on
  * which axis is a category axis.
  *
+ * Adding data with multiple occurences of a category to a series shows the last occurence.
```

Comment:
you probably want to add <p> on line 63

this PR does not require CSR as the change in javadoc is mere clarification rather than a substantial API change.